src/components/maps/controls/CustomControl.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, cloneElement } from 'react'
```

Comment:
I wanted to give an alternative approach to using the alertdialog also. I was curious about how to make a custom control and also use the control positions.

Reply:
@vnugent sorry I kind of let this pr sit for several weeks. Do you have any input on using a custom control versus the AlertDialog component like we had originally discussed? I'm fine either way, I have always been curious about how to make a maplibre custom control, so I explored doing that. 

in addition, I'm not sure why the preview is not working. I see the CI jobs are showing that docker-compose isn't found.

Reply:
I think you can use the existing AlertDialog.  We should do UI cleanup/overhaul at some point.
The Vercel preview works for me.  The docker-compose GitHub action may be broken. We can ignore it for now as long as Vercel build is green.

Reply:
@vnugent sounds good, I'll remove the custom control and use the AlertDialog component. 

In addition, I can't promise a time commitment (it's about to be climbing season 😄), but I would like to be involved in the ui cleanup effort.